acceptance/tests/reports/corrective_change_via_puppet.rb

```diff
@@ -0,0 +1,105 @@
+require 'yaml'
+
+test_name "C98094 - a resource changed via Puppet manifest will not be reported as a corrective change" do
+require 'puppet/acceptance/environment_utils'
+extend Puppet::Acceptance::EnvironmentUtils
```

Comment:
The above two lines should be indented to match the other elements of the block or moved above the test block with the other require.